src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -224,11 +220,11 @@ public void Properties_AddItemToDictionary_ItemPresent()
         }
 
         [OuterLoop("Uses external servers")]
-        [Theory, MemberData(nameof(EchoServers))]
-        public async Task SendAsync_SimpleGet_Success(Uri remoteServer)
+        [Theory, MemberData(nameof(RemoteServersMemberData))]
+        public async Task SendAsync_SimpleGet_Success(Configuration.Http.RemoteServer remoteServer)
         {
-            using (HttpClient client = CreateHttpClient())
-            using (HttpResponseMessage response = await client.GetAsync(remoteServer))
+            using (HttpClient client = CreateHttpClientForRemoteServer(remoteServer))
```

Comment:
This HttpClientHandlerTest class has derivations for both 1.1:
https://github.com/dotnet/corefx/blob/290a6e54a482ee950c4b812ad85f8de2d795932f/src/System.Net.Http/tests/FunctionalTests/SocketsHttpHandlerTest.cs#L944
and 2.0:
https://github.com/dotnet/corefx/blob/290a6e54a482ee950c4b812ad85f8de2d795932f/src/System.Net.Http/tests/FunctionalTests/SocketsHttpHandlerTest.cs#L2232

In conjunction with this remote server approach, that then means we're running the same tests more than we should be? e.g. for both the 1.1 and 2.0 derivations, we'll run both 1.1 and 2.0 tests?

Reply:
Yep. See #39345.

Reply:
Note also that since a bunch of the tests in HttpClientHandlerTest are 1.1-loopback based (not generic loopback), we're also running these tests twice for 1.1, since in the "2.0" derivation they still run as 1.1.

The "2.0" derivation really means "use the 2.0 loopback server". It would be nice to clarify this, but it's not urgent.